src/promptflow/promptflow/_sdk/_service/apis/span.py

```diff
@@ -0,0 +1,118 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from dataclasses import dataclass
+
+from flask_restx import fields
+
+from promptflow._constants import (
+    SpanAttributeFieldName,
+    SpanContextFieldName,
+    SpanFieldName,
+    SpanResourceAttributesFieldName,
+    SpanResourceFieldName,
+    SpanStatusFieldName,
+)
+from promptflow._sdk._service import Namespace, Resource
+from promptflow._sdk._service.utils.utils import get_client_from_request
+
+api = Namespace("spans", description="Span Management")
+
+# parsers for query parameters
+list_span_parser = api.parser()
+list_span_parser.add_argument("session_id", type=str, required=False)
+list_span_parser.add_argument("parent_span_id", type=str, required=False)
+
+
+# use @dataclass for strong type
+@dataclass
+class ListSpanParser:
+    session_id: str | None = None
```

Comment:
It seems not supported in python3.9
![image](https://github.com/microsoft/promptflow/assets/26239730/f78e18b8-21c8-4303-b9a7-a894d56c1153)

Reply:
good catch, will use `typing` here